<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>webrtc拉流播放demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="m_l_url">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>URL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_url">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>521</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_b_play">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>48</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
